packages/ui/src/variables.css

```diff
@@ -230,6 +237,22 @@
   --question-form-margin-top: var(--layout-margin-small);
   --walkthrough-footer-back-width: 44px;
   --walkthrough-footer-next-width: 228px;
+  --transition-medium: 300ms;
+  --theme-black-opacity-40: #00000040;
+  --walkthrough-card-max-width: 27.5rem;
+  --walkthrough-card-border-transparent: 1px solid #00000000;
+  --walkthrough-card-border-default: 1px solid
+    var(--surface-color-border-default);
+  --walkthrough-card-title-text-decoration-hidden: underline 1px #00000000;
+  --walkthrough-card-title-text-decoration-color-transition: underline 1px;
+  --walkthrough-card-title-text-underline-offset: 0.2em;
+  --step-tracker-heading-padding-left: var(--layout-padding-xlarge);
+  --step-tracker-padding-top: var(--layout-padding-xsmall);
+  --step-tracker-padding-left: var(--layout-padding-medium);
+  --step-tracker-padding: var(--step-tracker-padding-top)
+    var(--step-tracker-padding-left);
```

Comment:
No issue just wondering if it's your intent to do this going forward rather than just using the original variables?

Reply:
Yeah it was just because of how they had to change between mobile and desktop overall. I'm not sure I would do it this way unless I had to.